modern-cpp-font-lock.el

```diff
@@ -163,6 +163,65 @@ http://en.cppreference.com/w/cpp/language/integer_literal"
   :type 'boolean
   :group 'modern-c++-font-lock)
 
+;; Binary literal integers
```

Comment:
Do you mind to remove this comment? It does not more information than your below code.